e2e/app/component/new-notebook-modal.ts

```diff
@@ -52,12 +54,12 @@ export default class NewNotebookModal extends Modal {
     return Textbox.findByName(this.page, {name: 'Name:'});
   }
 
-  async python3Radiobutton(): Promise<RadioButton> {
+  python3Radiobutton(): RadioButton {
```

Comment:
Remove `async` because it's not async function. Here and few other places.

Reply:
There is a rule in eslint that will flag an async function that does not have an await in the function body